### PR TITLE
Frontend : implement one interface

### PR DESCRIPTION
**Pull request by ptitdam2001**
- Material UI implementation
- login page
- list of series

### CODE REVIEW

File: client/web/src/locales/de/user.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+	"signin": {
+		"title": "Connexion",
```
Comment (cquennouelle) [inline client/web/src/locales/de/user.json]: Das ist aber kein Deutsch hier. Es klinget mehr als Französisch, nein ?

File: client/web/src/locales/fr/user.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+	"signin": {
+		"title": "Connexion",
+		"email": "Adresse Email",
+		"password": "Mot de passe",
+		"action": "Se Connecter",
+		"forgotten_password_link": "Mot de passe oublié?",
```
Comment (cquennouelle) [inline client/web/src/locales/fr/user.json]: In french, put a ' ' before a '?'.

File: client/web/src/locales/ja/translation.json [hunk 3]
```diff
@@ -1,3 +1,8 @@
 {
+    "menu": {
+        "home": "Dashboard",
```
Comment (cquennouelle) [inline client/web/src/locales/ja/translation.json]: ス翻訳ステト ?

**Review by cquennouelle — COMMENTED** [review]
As you can see, I didn't understand anything in this code. It's not your fault, but mine.
Did write all those things or are they generated?

File: client/web/src/components/series/list/SeriesList.tsx [hunk 4]
```diff
@@ -1,24 +1,58 @@
...
-		</>
-	)
+  return (
+    <TableContainer component={Paper}>
+      <Table className={classes.table} aria-label="simple table">
+        <TableHead>
+          <TableRow>
+            <TableCell>seriesInstanceUID</TableCell>
+            <TableCell align="right">title</TableCell>
+            <TableCell align="right">title</TableCell>
```
Comment (bat3) [inline client/web/src/components/series/list/SeriesList.tsx]: Change with other name than title

File: client/web/src/components/users/Signin/SignIn.tsx [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+  const dispatch = useDispatch()
+  const history = useHistory()
+  const { t } = useTranslation()
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault()
+    // Call the API to login & returns informations about logged user
+    dispatch({
+      type: actionTypes.SET_CURRENT_USER,
+      user: { lastname: 'damien' },
```
Comment (bat3) [inline client/web/src/components/users/Signin/SignIn.tsx:35]: Is it really wanted ? If yes why ? ^^

File: client/web/src/locales/de/user.json (hunk 1 above)
Comment (ptitdam2001) [reply client/web/src/locales/de/user.json]: I never spoke german

File: client/web/src/components/users/Signin/SignIn.tsx (hunk 5 above)
Comment (ptitdam2001) [reply client/web/src/components/users/Signin/SignIn.tsx:35]: yes we must store the current user in store, but we will have to keep some user data like firstname, lastname, auth token (for api calls)

For the moment it is a fake

**Comment by ptitdam2001** [comment]
> As you can see, I didn't understand anything in this code. It's not your fault, but mine.
> Did write all those things or are they generated?

I wrote all those things, there are some web component using designed one (http://material-ui.com), styles

**Review by bat3 — APPROVED** [review]

**Pull request merged**